ethereumkit/src/test/java/org/p2p/ethereumkit/TokenBalancesTest.kt

```diff
@@ -26,34 +26,21 @@ class TokenBalancesTest : KoinTest {
 
     @get:Rule
     val koinTestRule = KoinTestRule.create {
+        allowOverride(true)
         printLogger(Level.DEBUG)
         modules(
-            EthereumNetworkModule.create(),
-            EthereumModule.create()
+            EthereumKitService.getEthereumKitModules(),
         )
+        module {
+            single<CoroutineDispatchers> { TestCoroutineDispatcher() }
+        }
     }
 
     @Test
     fun getBalance() = runTest {
-        val balance = repository.getTokenBalances(tokenKeyProvider.address)
-        val stringBuilder = StringBuilder()
-        balance.balances.map { it.contractAddress }.forEach { address ->
-            val tokenMetadata = repository.getTokenMetadata(
-                contractAddresses = address
-            )
-            val tokenName = tokenMetadata.tokenName
-            val tokenSymbol = tokenMetadata.symbol
-            val tokenDecimals = tokenMetadata.decimals
-            val tokenLogo = tokenMetadata.logoUrl
-            val tokenBalance = balance.balances.first { it.contractAddress == address }.tokenBalance
-            stringBuilder.append("Token Name: $tokenName ")
-            stringBuilder.append("Token Address: $address ")
-            stringBuilder.append("Token Symbol: $tokenSymbol ")
-            stringBuilder.append("Token Balance: $tokenBalance ")
-            stringBuilder.append("Token Decimals: $tokenDecimals ")
-            stringBuilder.append("Token logo: $tokenLogo ")
-            stringBuilder.append("\n")
-        }
-        println(stringBuilder)
+        repository.init(seedPhrase = "apart approve black comfort steel spin real renew tone primary key cherry".split(" "))
+        val result = repository.loadWalletTokens()
+        assert(result.isNotEmpty())
```

Comment:
`org.assertj.core.api.Assertions.assertThat(result).isNotEmpty`

Reply:
I have not this dependencies in my module x D